src/patients/GeneralInformation.tsx

```diff
@@ -273,27 +246,24 @@ const NewPatientForm = (props: Props) => {
               name="address"
               value={patient.address}
               isEditable={isEditable}
-              onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
-                onFieldChange('address', event.currentTarget.value)
-              }}
+              onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) =>
+                onFieldChange && onFieldChange('address', event.currentTarget.value)}
             />
           </div>
         </div>
-        {isEditable && (
-          <div className="row float-right">
-            <div className="btn-group btn-group-lg">
-              <Button className="mr-2" color="success" onClick={onSaveButtonClick}>
-                {t('actions.save')}
-              </Button>
-              <Button color="danger" onClick={() => onCancel()}>
-                {t('actions.cancel')}
-              </Button>
-            </div>
```

Comment:
I think we want to keep these buttons

Reply:
that's where the buttons were in NewPatientForm, the entire file is deleted but Github thinks I renamed it to GeneralInformation and deleted GeneralInformation. The buttons are still there, they're just slightly further down. GeneralInformation has a different layout than NewPatientForm because it has the panels.

Reply:
ok, I restored the previous style of buttons